evap/evaluation/tests/test_models.py

```diff
@@ -670,6 +670,19 @@ def test_get_sorted_due_evaluations(self):
         sorted_evaluations = student.get_sorted_due_evaluations()
         self.assertEqual(sorted_evaluations, [(evaluations[1], 0), (evaluations[0], 0), (evaluations[2], 1)])
 
+    def test_can_be_sorted(self):
```

Comment:
I mean, they _can_ be sorted, but _are_ they? And is that sorting correct? Maybe `test_correct_sorting`?